docs/spec/broker.md

```diff
@@ -40,10 +40,28 @@ default Broker upon creation if no Broker is specified by the user.
 A Trigger MAY be created before its assigned Broker exists. A Trigger SHOULD
 progress to Ready when its assigned Broker exists and is Ready.
 
+#### Event filters
+
+Implementations of Trigger MUST support both attributes filter and expression
+filter.
+
 The attributes filter specifying a list of key-value pairs MUST be supported by
 Trigger. Events that pass the attributes filter MUST include context or
 extension attributes that match all key-value pairs exactly.
 
+The expression filter specifying a Javascript expression MUST be supported by
```

Comment:
Can you give a couple examples of javascript expressions? I am not familiar with this.

Reply:
Added in the doc